app/src/main/java/ch/epfl/sweng/eventmanager/ui/eventSelector/EventPickingActivity.java

```diff
@@ -164,7 +166,18 @@ public boolean onPrepareOptionsMenu(Menu menu) {
 
     @Override
     public void onBackPressed() {
-        finish();
-        super.onBackPressed();
+        if (doubleBackToExitPressedOnce) {
+            finish();
+            return;
+        }
+        this.doubleBackToExitPressedOnce = true;
+        Toast toast = Toast.makeText(this, R.string.double_back_press_to_exit, Toast.LENGTH_SHORT);
+        ((TextView) (toast.getView().findViewById(android.R.id.message))).setTextColor(ContextCompat.getColor(this,
+                R.color.colorSecondary));
+        toast.getView().getBackground().setColorFilter(ContextCompat.getColor(this, R.color.colorPrimary),
```

Comment:
Is it possible to set the Color in an xml file ?

Reply:
You need to create a custom layout for the toast if you want to do that but I found this introduces a lot of complexity  in the just to customize the colors in a toast so that's why I chose that solution instead :)